modelbuilders_bench/lgbm_mb.py

```diff
@@ -99,7 +98,7 @@
     if 'cudf' in str(type(y_train)):
         params.n_classes = y_train[y_train.columns[0]].nunique()
     else:
-        params.n_classes = len(np.unique(y_train))
+        params.n_classes = int(max(y_train)) + 1
```

Comment:
I guess it's safer to add extra inner `if` for case where max of uniques is not equal to length of uniques - 1